test/suite/test_rollback_to_stable17.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
```

Comment:
Create a follow-on ticket to add the support of the column store to all the existing RTS tests. As part of this ticket, Add the scenario of the `in-memory` database also.

Reply:
I created WT-7269 to cover these changes.